containers-tests/tests/intmap-properties.hs

```diff
@@ -237,36 +246,43 @@ tests = [ testGroup "Test Case" [
 -- Operators
 
 test_index :: Assertion
-test_index = fromList [(5,'a'), (3,'b')] ! 5 @?= 'a'
+test_index = do
+    fromList [(5,'a'), (-3,'b')] ! 5 @?= 'a'
+    fromList [(5,'a'), (-3,'b')] ! (-3) @?= 'b'
```

Comment:
Are these used anywhere?

Reply:
What do you mean with "these", "used" and "anywhere"?

Reply:
I mean I don't see anything actually *running* this test or the one below it.

Reply:
Aren't they literally the first two tests in the `main` of the `intmap-properties.hs` file?

Reply:
Yes, I am reading stupid.

Reply:
The original test is still not here. I'm guessing you tried to restore it and misread/mistyped.